CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 100.0.2
```

Comment:
Shouldn't this be `100.1.0`?  This isn't a bug fix, but a change in functionality. Not reading the hint for a Layout change notification isn't a bug - it's how UIAccessibility is intended to work.  Snackbars aren't "major" portions of the screen, so it's not really a bug to have used LayoutChanged previously.

Reply:
Based on the internal a11y team's feedback, including input from people who need to use VoiceOver on a daily basis, it seems like the hint should be read in this case, making it a bug from the accessibility perspective.

Furthermore, us having an API for accessibilityHint but it not being announced sounds like an issue by its own right.

In my opinion this is definitely a bug fix, but it is also a break in behavior. I am not sure if this warrants a major number change due to its breaking behavior.

Reply:
To be clear, I was advocating for a minor version bump due to an improvement/feature change. Last I checked, we only major version bump if we break the build process, not functionality.

Reply:
Done